web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId].tsx

```diff
@@ -58,6 +60,19 @@ const DeploymentDetailsPage = (props: DeploymentDetailsPageProps) => {
     onWsError,
   })
 
+  const onCopyDeployment = async () => {
+    const url = await actions.onCopyDeployment()
+    if (!url) {
+      return
+    }
+
+    router.push(url)
+  }
+
+  const anchors = useAnchorActions({
+    copyDeployment: () => onCopyDeployment(),
```

Comment:
this should be just copyDeployment: onCopyDeployment. or even better: just rename the onCopyDeployment function